src/libserialize/collection_impls.rs

```diff
@@ -20,6 +20,7 @@ use std::sync::Arc;
 impl<
     T: Encodable
 > Encodable for LinkedList<T> {
+    #[inline]
```

Comment:
I thought `#[inline]` was unnecessary on generic methods?

Reply:
Here's the discussion I remembered: https://github.com/rust-lang/rust/pull/52704